internal/provider/resource_group.go

```diff
@@ -118,9 +118,15 @@ func resourceGroupCreate(ctx context.Context, d *schema.ResourceData, meta inter
 		}
 		gcsmr, err := grps.SetMembers(ctx, gcr.Item.Id, gcr.Item.Version, memberIds)
 		if err != nil {
+			if err := setFromGroupResponseMap(d, raw); err != nil {
+				return diag.FromErr(err)
```

Comment:
This clobbers the original error, is there some way we can just log this error instead and show the original error to the user?

Reply:
good catch, maybe I can do something like this?
diag.Errorf("%s: failed to set resource state %s", originalErr.Error(), setStateErr.Error())

Reply:
Sounds good!